Core/HLE/sceMp3.cpp

```diff
@@ -167,10 +165,12 @@ static int sceMp3Decode(u32 mp3, u32 outPcmPtr) {
 		
 	int pcmBytes = ctx->AuDecode(outPcmPtr);
 	if (pcmBytes > 0) {
+		DEBUG_LOG(ME, "%d = sceMp3Decode(%08x,%08x)", pcmBytes, mp3, outPcmPtr);
 		// decode data successfully, delay thread
 		return hleDelayResult(pcmBytes, "mp3 decode", mp3DecodeDelay);
 	}
-	return pcmBytes;
+	ERROR_LOG(ME, "%d = sceMp3Decode(%08x,%08x)", 0x807f00fd, mp3, outPcmPtr);
+	return ERROR_AVCODEC_INVALID_DATA;
```

Comment:
The only negative case right now would be invalid address:
https://github.com/hrydgard/ppsspp/blob/b76e906ed0960a4a352ad9eb291cf0558637193b/Core/HW/SimpleAudioDec.cpp#L332

This probably won't change anything.  If we're seeing the JpcspTrace return an error it may mean we need better validation.

-[Unknown]

Reply:
JPCSP also don't have have this error code in sceMp3Decode.
I should wait it first.

Reply:
According to this http://lukasz.dk/mirror/forums.ps2dev.org/viewtopic108b.html?t=10798
Error 0x807F00FD might be related to ID3 Tag
![image](https://user-images.githubusercontent.com/7974720/106352679-9b8c6880-6317-11eb-864e-687d6cff2fa5.png)